src/pages/guidelines/color/overview.mdx

```diff
@@ -261,54 +289,230 @@ attention or create visual tension.
   </DoDont>
 </DoDontRow>
 
+<br />
+
+### Tokens
+
+Color in Carbon is managed through tokens. Tokens are an abstracted method of
+applying color in a consistent and scalable way. Each color token is assigned a
+role and a value. The role determines where the token is applied in the UI and
+the value is the actual color (that is, the hex code) that will appear in an
+assigned theme. Tokens are used in place of a one-off hex code, allowing for
+changes in values to be made at scale for easy color management. The token names
+themselves stay the same in code no matter which theme is being used.
+
+Here's an example of how tokens work: instead of coding all instances of input
```

Comment:
Loving the example with form, when reading it I wonder if it'd help to explain this in reverse order. In other words, show the problem and show how tokens help to solve it.

Maybe something like:

> When you start building a component or designing a part of your product, you may be tempted to use colors from the IBM Design Language directly. For example, you may be building out a form and want all of the input labels to be the same color. If you're working in a lighter theme, you could decide to use `#565656` for these colors and this value will be used in code.
>
> In the future, your product may need to be updated to instead use a darker theme. Or, you may want to support a user preference of light mode and dark mode in your product. In this case, you need the color of the labels in your UI to be `#ffffff`.
>
> Tokens help us abstract how we use color from the values themselves. In this situation, you would use the `$text-secondary` token instead of providing a hex color directly. Under the hood, the `$text-secondary` token will dynamically map to `#565656` or `#ffffff` depending on your theme.

Just a thought though 👀

Reply:
Then is when I miss Jan. I like the general sentiment but your example is written in the second person and most of the time (and with the rest of this page) we write in third person. I've been trying to write a pass in the thrid person and its harder then it seems. I'm going to just leave this for now but hopefully come back to it when I have a little more time.